android/src/com/frostwire/android/gui/fragments/preference/SearchFragment.java

```diff
@@ -141,4 +151,33 @@ private void checkAllEngines(boolean checked) {
             UIUtils.showShortMessage(getView(), R.string.search_preferences_one_engine_checked_always);
         }
     }
+
+    @Override
+    protected RecyclerView.Adapter onCreateAdapter(PreferenceScreen preferenceScreen) {
+        return new CheckedAwarePreferenceGroupAdapter(preferenceScreen, R.color.selected_search_background, R.color.basic_white);
+    }
+
+    private class CheckedAwarePreferenceGroupAdapter extends PreferenceGroupAdapter {
+
+        final int checkedDrawableId;
+        final int unCheckedDrawableId;
+
+        CheckedAwarePreferenceGroupAdapter(PreferenceGroup preferenceGroup, int checkedDrawableId, int unCheckedDrawableId) {
+            super(preferenceGroup);
+
+            this.checkedDrawableId = checkedDrawableId;
+            this.unCheckedDrawableId = unCheckedDrawableId;
+        }
+
+        @Override
+        public void onBindViewHolder(PreferenceViewHolder holder, int position) {
+            final CheckBoxPreference preference = (CheckBoxPreference) getItem(position);
+
+            preference.onBindViewHolder(holder);
+            if (!preference.getKey().equals("frostwire.prefs.search.preference_category.select_all")) {
```

Comment:
isn't there a `Constants.PREF_KEY_...` value for this?
There should be one, we don't usually have preferences like this hardcoded in the code.
If you find any, let's make sure we use `Constants.PREF_KEY_...` values